sound/soc/sof/intel/mtl.c

```diff
@@ -0,0 +1,920 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// Copyright(c) 2022 Intel Corporation. All rights reserved.
+//
+// Authors: Ranjani Sridharan <ranjani.sridharan@linux.intel.com>
```

Comment:
You can also claim credit for yourself ;)

Reply:
> You can also claim credit for yourself ;)

That's not necessary, most code comes from Ranjani. :)